leo_fw/test_data/encoder.yaml

```diff
@@ -0,0 +1,20 @@
+test1:
+  velocity: 1.0
+  time: 2
+  offset: 0.2
+test2:
+  velocity: 2.0
+  time: 2
+  offset: 0.2
+test3:
+  velocity: 3.0
+  time: 2
+  offset: 0.2
+test4:
+  velocity: 4.0
+  time: 3
+  offset: 0.2
```

Comment:
Instead of defining each test as a different yaml struct, you can define one array of structs. It would look like this:
```yaml
tests:
- velocity: 1.0
  time: 2
  offset: 0.2
- velocity: 2.0
  time: 2
  offset: 0.2
- velocity: 3.0
  time: 2
  offset: 0.2
- velocity: 4.0
  time: 3
  offset: 0.2
```
or like this:
```yaml
tests:
- {velocity: 1.0, time: 2, offset: 0.2}
- {velocity: 2.0, time: 2, offset: 0.2}
- {velocity: 3.0, time: 2, offset: 0.2}
- {velocity: 4.0, time: 3, offset: 0.2}
```